support-frontend/assets/pages/digital-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -312,69 +309,30 @@ function CheckoutForm(props: PropTypes) {
-                      text="Credit/Debit card"
-                      name="paymentMethod"
-                      checked={props.paymentMethod === 'Stripe'}
-                      onChange={() => props.setPaymentMethod('Stripe')}
-                    />
-                  </Fieldset>
-                </div>
-              }
-                <div>
-                  <Text>
-                    <p>
-                      <strong>Money Back Guarantee.</strong>
-                      If you wish to cancel your subscription, we will send you
-                      a refund of the unexpired part of your subscription.
-                    </p>
-                    <p>
-                      <strong>Cancel any time you want.</strong>
-                      There is no set time on your agreement so you can stop
-                      your subscription anytime
-                    </p>
-                  </Text>
-                </div>
-              </Rows>
-              <DirectDebitPopUpForm
-                onPaymentAuthorisation={(pa: PaymentAuthorisation) => {
-                  props.onPaymentAuthorised(pa);
-                }}
-              />
-            </FormSection>
+            <PaymentMethodSelector
+              countrySupportsDirectDebit={props.countrySupportsDirectDebit}
+              paymentMethod={props.paymentMethod}
+              setPaymentMethod={props.setPaymentMethod}
+              onPaymentAuthorised={props.onPaymentAuthorised}
+            />
             <FormSection>
               {errorState}
-              {props.paymentMethod === 'PayPal' ? (
-                <PayPalExpressButton
-                  onPaymentAuthorisation={props.onPaymentAuthorised}
-                  csrf={props.csrf}
-                  currencyId={props.currencyId}
-                  hasLoaded={props.payPalHasLoaded}
-                  canOpen={props.formIsValid}
-                  onClick={props.validateForm}
-                  formClassName="form--contribution"
-                  isTestUser={props.isTestUser}
-                  setupRecurringPayPalPayment={props.setupRecurringPayPalPayment}
-                  amount={props.amount}
-                  billingPeriod={props.billingPeriod}
-                />
-              ) : (
-                <Button aria-label={null} type="submit">Continue to payment</Button>
-              )}
+              <SubscriptionSubmitButtons
+                paymentMethod={props.paymentMethod}
+                onPaymentAuthorised={props.onPaymentAuthorised}
+                csrf={props.csrf}
+                currencyId={props.currencyId}
+                payPalHasLoaded={props.payPalHasLoaded}
+                formIsValid={props.formIsValid}
+                validateForm={props.validateForm}
+                isTestUser={props.isTestUser}
+                setupRecurringPayPalPayment={props.setupRecurringPayPalPayment}
+                amount={props.amount}
+                billingPeriod={props.billingPeriod}
```

Comment:
might be cleaner to spread props here?

Reply:
Actually there are so many props in `PropTypes` that are not in `SubscriptionSubmitButtons` I don't think this will work. Unless you have a cunning way of doing it?